conformance/smart-config-fhirr4-service/utils_generator.bal

```diff
@@ -24,11 +23,10 @@ import ballerina/url;
 # + discoveryEndpoint - Discovery endpoint
 # + return - If successful, returns OpenID configuration as a json. Else returns error.
 public isolated function getOpenidConfigurations(string discoveryEndpoint) returns OpenIDConfiguration|error {
-    log:printDebug("Retrieving openid configuration started");
+    log:printInfo("Retrieving openid configuration started");
```

Comment:
this should be a debug log?

Reply:
Fixed in https://github.com/wso2/open-healthcare-prebuilt-services/commit/6ba8462e30e7915f54f620e1f211b28efb5d6e55